SocialCareCaseViewerApi.Tests/V1/Factories/ResponseFactoryTests.cs

```diff
@@ -358,10 +358,10 @@ public void CanMapPersonDetailsToGetPersonResponse()
                 disability2
             };
 
-            person.OtherEmails = new List<OtherEmail>
+            person.Emails = new List<EmailAddress>
```

Comment:
we've done this as a list, as `phoneNumbers` is, right?

Reply:
Yes
```
"emails": [
  {
   "email": "foo.bar@gmail.com",
   "type": "test"
  },
  {
   "email": "foo.bar@gmail.com",
   "type": "xxx"
  }
  ],
```